features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Resources/Resources.jsx

```diff
@@ -500,14 +558,24 @@ class Resources extends React.Component {
         }
     };
     toggleAddResource = () => {
-        this.setState({ showAddResource: !this.state.showAddResource, showScopes: false });
+        this.setState({ showAddResource: !this.state.showAddResource });
+    }
+    toggleAssignPolicy = () => {
+        this.setState({ showPolicy: !this.state.showPolicy, showAddResource: false });
     }
-    toggleAssignScopes = () => {
-        this.setState({ showScopes: !this.state.showScopes, showAddResource: false });
+    advancedPolicyTypeChange = (event) => {
+        if('perResource' === event.target.value) {
+        } else if('perAPI' === event.target.value) {
+            this.setState({ selectedPolicy: this.props.api.apiThrottlingPolicy });
```

Comment:
minor formatting issue in `if` and `else if`.
use destructuring for `this.props.api.apiThrottlingPolicy`